src/service/viewport/viewport-binding-ios-embed-sd.js

```diff
@@ -396,15 +396,14 @@ export class ViewportBindingIosEmbedShadowRoot_ {
 
   /** @override */
   getContentHeight() {
-    // The reparented body inside scroller will have the correct content height.
-    // Body is overflow: hidden so that the scrollHeight include the margins of
-    // body's first and last child.
-    // Body height doesn't include paddingTop on the parent, so we add on the
-    // position of the body from the top of the viewport and subtract the
-    // scrollTop (as position relative to the viewport changes as you scroll).
-    return this.wrapper_./*OK*/scrollHeight
+    const scrollingElement = this.wrapper_;
+    const rect = scrollingElement./*OK*/getBoundingClientRect();
```

Comment:
Consider adding a comment explaining why we don't use `scrollHeight` anymore. E.g.

```js
// Don't use scrollHeight since it always returns MAX(viewport_height, document_height)
// even though we want the latter. Also, it doesn't account for margins.
```